src/containers/Bookings.js

```diff
@@ -1,19 +1,79 @@
 import React, { Component } from "react";
-import Search from "../components/Search.js";
-// import Results from "../components/Results.js";
-// import * as FakeBookings from "../data/fakeBookings.json";
+import Search from "../components/Search/Search.js";
```

Comment:
I like that you've organised your components into sub-directories!

Reply:
Thank you.